components/collector/src/source_collectors/harbor/security_warnings.py

```diff
@@ -98,24 +122,6 @@ async def _create_entity(self, artifact: dict) -> Entity:
             fixable=scan_summary.get("fixable", 0),
         )
 
-    def _include_entity(self, entity: Entity) -> bool:
```

Comment:
We built this in #4714 and #5240. I think it makes sense to revise the `collect>parse>include` flow to something like `collect>filter>parse>include`, so we have a generic way to implement such pre-work filtering. Some passthrough method in `base_collectors/source_collector.py`?